extension/WebJobs.Extensions.RabbitMQ/Bindings/RabbitMQAsyncCollector.cs

```diff
@@ -17,16 +17,14 @@ internal class RabbitMQAsyncCollector : IAsyncCollector<ReadOnlyMemory<byte>>
         public RabbitMQAsyncCollector(RabbitMQContext context, ILogger logger)
         {
             _logger = logger ?? throw new ArgumentNullException(nameof(logger));
-            _context = context ?? throw new ArgumentNullException(nameof(context));
-            if (_context.Service == null)
-            {
-                throw new ArgumentNullException("context.service");
-            }
+            _ = context ?? throw new ArgumentNullException(nameof(context));
+            _ = context.Service ?? throw new ArgumentException("Value cannot be null. Parameter name: context.Service");
+            _context = context;
```

Comment:
These are all changes to fix code quality and code style analysis reported issues.